DGC.ValueSets.schema.json

```diff
@@ -6,105 +6,95 @@
-        { "code": { "const": "ORG-100024420"}},
-        { "code": { "const": "ORG-100032020"}},
-        { "code": { "const": "Gamaleya-Research-Institute"}},
-        { "code": { "const": "Vector-Institute"}},
-        { "code": { "const": "Sinovac-Biotech"}},
-        { "code": { "const": "Bharat-Biotech"}}
+      "type": "string",
+      "enum": [
+        "ORG-100001699",
+        "ORG-100030215",
+        "ORG-100001417",
+        "ORG-100031184",
+        "ORG-100006270",
+        "ORG-100013793",
+        "ORG-100020693",
+        "ORG-100020693",
+        "ORG-100010771",
+        "ORG-100024420",
+        "ORG-100032020",
+        "Gamaleya-Research-Institute",
+        "Vector-Institute",
+        "Sinovac-Biotech",
+        "Bharat-Biotech"
       ]
     },
     "test-manf" : {
       "description": "EU eHealthNetwork: Value Sets for Digital Green Certificates. version 1.0, 2021-04-16, section 2.8",
-      "type": "object",
-      "required": ["code"],
-      "oneOf": [
-        { "code": {"const": "1232" }},
-        { "code": {"const": "1304"}},
-        { "code": {"const": "1065" }},
-        { "code": {"const": "1331" }},
-        { "code": {"const": "1484" }},
-        { "code": {"const": "1242" }},
-        { "code": {"const": "1223" }},
-        { "code": {"const": "1173" }},
-        { "code": {"const": "1244" }},
-        { "code": {"const": "1360" }},
-        { "code": {"const": "1363" }},
-        { "code": {"const": "1767" }},
-        { "code": {"const": "1333" }},
-        { "code": {"const": "1268" }},
-        { "code": {"const": "1180" }},
-        { "code": {"const": "1481" }},
-        { "code": {"const": "1162" }},
-        { "code": {"const": "1271" }},
-        { "code": {"const": "1341" }},
-        { "code": {"const": "1097" }},
-        { "code": {"const": "1489" }},
-        { "code": {"const": "344"  }},
-        { "code": {"const": "345"  }},
-        { "code": {"const": "1218" }},
-        { "code": {"const": "1278" }},
-        { "code": {"const": "1343" }}
-      ]
-    },
+      "type": "string",
+      "enum": [
+        "1232",
+        "1304",
+        "1065",
+        "1331",
+        "1484",
+        "1242",
+        "1223",
+        "1173",
+        "1244",
+        "1360",
+        "1363",
+        "1767",
+        "1333",
+        "1268",
+        "1180",
+        "1481",
+        "1162",
+        "1271",
+        "1097",
+        "1489",
+        "344",
+        "345",
+        "1218",
+        "1343"
+      ]    },
     "test-result": {
       "description": "EU eHealthNetwork: Value Sets for Digital Green Certificates. version 1.0, 2021-04-16, section 2.9",
-      "type": "object",
-      "required": ["code"],
-      "oneOf": [
-        { "code": { "const": "260415000"}},
-        { "code": { "const": "260373001"}}
+      "type": "string",
+      "enum": [
+        "260415000",
+        "260373001"
       ]
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Interesting one: by habit I always add blank newlne (been in ISO standard for C for decades and also C++, is also in PEP8 for python) but interestingly enough two different code formatters for JSON actually *removed* the blank line at end-of-file. 
Maybe check this one against spec / JSON formatters?

Reply:
I'd recommend that we just reformat using `jq` or similar before commit (and check in the CI pipeline of course). I'd be happy to fix this once we move close to the target.

Reply:
All using it shoudl work with the same formatter, otherwise we'll end up with seemingly endless add blank line / remove blank line "changes"
Maybe sth to take up the README? No particular preference from my side on a formatting tool, so I'm fine with jq if that works for you.